core/frontend/vite.config.js

```diff
@@ -48,7 +48,7 @@ export default defineConfig({
       version: 2.7,
     }),
   ],
-  assetsInclude: ['**/*.gif', '**/*.glb', '**/*.png', '**/*.svg'],
+  assetsInclude: ['**/*.gif', '**/*.glb', '**/*.png', '**/*.svg', '**/assets/ArduPilot-Parameter-Repository**.json'],
```

Comment:
Did you test this through a docker release ?

Reply:
yep